src/installer/corehost/cli/hostpolicy/standalone/coreclr_resolver_t.cpp

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <cassert>
+#include <error_codes.h>
+#include <utils.h>
+#include "coreclr_resolver_t.h"
```

Comment:
In the installer subset, the name of types typically end with `_t`  but the filenames do not include the `_t`.
So, this file should be called `coreclr_resolver.cpp`

Reply:
from another code review:

> The class name must match the file name. So, the class should be called hostfxr_iface_t.
However, in keeping with the hostpolicy_resolver naming, this class (and file) should probably be called hostfxr_resolver_t.

I will change, but this is a confusing policy to follow.

Now we will have `coreclr_resolver.h` next to `hostfxr_resolver_t.h` . . .

Reply:
Can you point me to the PR with that comment? In general I absolutely agree that file name and class name should match, but the overriding principle has been "match the style of the code around". Unfortunately the host components have this weird style of calling types with `_t` suffix, but not including that in the file name. So the file names should be `coreclr_resolver.h` and `hostfxr_resolver.h`.

(If somebody wants to pick up a side project of changing the host to a naming scheme which is closer to the rest of our code base I would be all for it).

Reply:
That was in https://github.com/dotnet/runtime/pull/35368#discussion_r414926293

So, I hear - we should rename `hostfxr_resolver_t.h` to `hostfxr_resolver.h`

I am not sure it is a good time to fix the naming scheme. Perhaps later.